test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -279,8 +286,9 @@ run_test_checkpoint()
     echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
     echo "Running test checkpoint in branch: \"$branch_name\""
     echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
-
+    pwd;
```

Comment:
Just hope to confirm it's an intentional change, instead of a debugging leftover.

Reply:
Ah, this is a debugging leftover. Thanks for spotting that!